Description: Disable JIT on ppc64el, needs explicit porting to ELFv2.
Author: Adam Conrad <adconrad@ubuntu.com>

--- a/sljit/sljitConfigInternal.h
+++ b/sljit/sljitConfigInternal.h
@@ -109,7 +109,11 @@
 #elif defined (__aarch64__)
 #define SLJIT_CONFIG_ARM_64 1
 #elif defined(__ppc64__) || defined(__powerpc64__) || defined(_ARCH_PPC64) || (defined(_POWER) && defined(__64BIT__))
-#define SLJIT_CONFIG_PPC_64 1
+# if _CALL_ELF != 2
+#  define SLJIT_CONFIG_PPC_64 1
+# else
+#  define SLJIT_CONFIG_UNSUPPORTED 1
+# endif
 #elif defined(__ppc__) || defined(__powerpc__) || defined(_ARCH_PPC) || defined(_ARCH_PWR) || defined(_ARCH_PWR2) || defined(_POWER)
 #define SLJIT_CONFIG_PPC_32 1
 #elif defined(__mips__) && !defined(_LP64)
